docs/quickstart/introduction.rst

```diff
@@ -0,0 +1,120 @@
+Introduction
+============
+
+Deploying a MetalK8s cluster, while simple, will be facilitated by
+an introductory explanation of a few concepts and an outline of the deployment
+procedure that will ensue.
+
+This guide will cover many aspects while not diving too much into the details.
+Links to relevant sections of the complete documentation are included when
+useful, and a :doc:`/glossary` is provided.
+
+Concepts
+^^^^^^^^
+Although being familiar with
+`Kubernetes concepts <https://kubernetes.io/docs/concepts/>`_
+is recommended, the necessary concepts to grasp before installing a MetalK8s
+cluster are presented here.
+
+Control-plane and workload-plane
+""""""""""""""""""""""""""""""""
+
+.. _`API Server`:
+      https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/
+.. _Scheduler:
+      https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/
+.. _`Controller Manager`:
+      https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/
+.. _etcd: https://etcd.io/
+
+
+This dichotomy is central to MetalK8s, and often referred to in other
+Kubernetes concepts.
+
+The **control-plane** is the set of machines (called :term:`nodes <Node>`) and
+the services running there that make up the essential Kubernetes functionality
+for running containerized applications, managing declarative objects, and
+providing authentication/authorization to end-users as well as services.
+The main components making up a Kubernetes control-plane are:
+
+- `API Server`_
+- Scheduler_
```

Comment:
Why `Scheduler` is not surrounded by backquote?

Reply:
They are not needed, only when the link contains spaces or special characters. I can add them for consistency if you'd like.

Reply:
I would do it for consistency yeah.